_pages/openid-connect.md

```diff
@@ -72,7 +72,7 @@ Users need to authorize OpenID Connect 1.0 clients individually. To present the
 
 ### Authorization request
 
-View example as <button data-example="pkce">PKCE</button><button data-example="private_key_jwt">private_key_jwt</button>
+View example as <button data-example="pkce">PKCE</button> or&nbsp;&nbsp; <button data-example="private_key_jwt">private_key_jwt</button>
```

Comment:
the hardcoded nbsp seems a little janky -- is that solving a display problem better solved with CSS?